app/assets/javascripts/dashboard.js

```diff
@@ -0,0 +1,27 @@
+document.addEventListener('DOMContentLoaded', () => {
+  // Get all "navbar-burger" elements
+  const $navbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.navbar-burger'), 0);
```

Comment:
don't use jquery-like variable names for javascript elements
could be very confusing